benchmarks/experiment_runner.py

```diff
@@ -144,20 +143,17 @@ def generate_and_run_all_configs(self):
         except subprocess.TimeoutExpired as e:
           self._fwd_captured_stdout_stderr(e.stdout, e.stderr)
           logger.error("TIMEOUT")
-          self.save_results(benchmark_experiment, benchmark_model,
-                            {"error": str(e)}, None)
+          self._save_results(experiment_cfg, model_cfg, {"error": str(e)})
         except subprocess.CalledProcessError as e:
           self._fwd_captured_stdout_stderr(e.stdout, e.stderr)
```

Comment:
Should this line be also under the last two exceptions?

Reply:
I think no because the last two exceptions mean that no subprocess was launched, so there would not be anything to fwd.